website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -135,3 +135,30 @@ Protect your tokens carefully because they contain the same permissions as your
 To revoke a token, click the **trash can** next to it. That token will no longer be able to authenticate as your user account.
 
 -> **Note**: When SAML SSO is enabled there is a session timeout for user API tokens, forcing users to periodically reauthenticate through the web UI in order to keep their tokens active. Refer to [API Token Expiration](/terraform/enterprise/saml/login#api-token-expiration) for details.
+
+
+### GitHub App OAuth Token
+
+Click **Tokens** in the sidebar to manage your GitHub App token. This token is required for tasks such as connecting workspaces to an available GitHub App installation.
+
+~> **Important:** Only a Terraform Cloud User can own a GitHub App token. Team and Organization API tokens are not able to own a GitHub App token.
+
+A GitHub App token is required for the following tasks:
+- Connecting workspaces, policy sets, and registry modules to a GitHub App installation with the [Terraform Cloud API](/terraform/cloud-docs/api-docs) and UI.
+- View available GitHub App installations with the [Terraform Cloud API](/terraform/cloud-docs/api-docs) and UI.
```

Comment:
```suggestion
A GitHub App token lets you:
- Connect workspaces, policy sets, and registry modules to a GitHub App installation with the [Terraform Cloud API](/terraform/cloud-docs/api-docs) and UI.
- View available GitHub App installations with the [Terraform Cloud API](/terraform/cloud-docs/api-docs) and UI.
```

Reply:
Also, should we move this information to the top of the page since it will help the reader decide if they should read on, and is a little bit of duplicated material from the first paragraph?

Reply:
Yeah I think that would make sense as well 👍